test/functional/feature_shutdown.py

```diff
@@ -20,8 +20,14 @@ def set_test_params(self):
 
     def run_test(self):
         node = get_rpc_proxy(self.nodes[0].url, 1, timeout=600, coveragedir=self.nodes[0].coverage_dir)
+        # Force connection establishment by executing a dummy command.
+        node.getblockcount()
```

Comment:
Is this required given that we wait anyway until node.waitfornewblock is called, which in turn should set up the connection?

Reply:
For the test to pass this is not required, but this is here to note that there's still a race after connection establishment. Let me know if it's not important.